src/ioctl.c

```diff
@@ -343,8 +343,9 @@ static long scoutfs_ioc_release(struct file *file, unsigned long arg)
 		}
 	}
 
-out:
+unlock:
 	scoutfs_unlock(sb, lock, SCOUTFS_LOCK_WRITE);
```

Comment:
And I don't think this is needed because the lock is NULL if it's unlocked and _unlock will not have a problem with it.